### PR TITLE
feature/LEAF-1760-2: Align Card Heights in User Groups and Service Chiefs

### PR DESCRIPTION
**Pull request by cameron-chilton**
Redo of original branch due to merge issues with mod_groups.tpl. Cherry picked new PR.

Changes made:
- Card heights aligned on User Groups and Service Chiefs by adding user count feature
- Added flexDisplayRow to Service Chiefs
- Added toTitleCase function to put names in title case
- Reversed names to be Firstname Lastname
- Cosmetic changes to Email Template Editor and Template Editor

### CODE REVIEW

**Review by jeremyrl — APPROVED** [review]

File: LEAF_Request_Portal/admin/templates/mod_groups.tpl [hunk 1]
```diff
@@ -206,16 +214,17 @@ function setPrimaryAdmin(userID) {
...
-        }
+        },
+        cache: false
     });
 }
 
 function focusGroupsAndMembers(groupID) {
     $('#' + groupID).on('focusin', function() {
-        $('#' + groupID).css('background-color', '#fffdc2');
+        //$('#' + groupID).css('background-color', '#fffdc2');
```
Comment (abiola-adewuyi) [inline LEAF_Request_Portal/admin/templates/mod_groups.tpl:224]: Remove unused code.

File: LEAF_Request_Portal/admin/templates/mod_groups.tpl [hunk 2]
```diff
@@ -206,16 +214,17 @@ function setPrimaryAdmin(userID) {
...
     });
     $('#' + groupID).on('focusout', function() {
-        $('#' + groupID).css('background-color', 'white');
+        //$('#' + groupID).css('background-color', 'white');
```
Comment (abiola-adewuyi) [inline LEAF_Request_Portal/admin/templates/mod_groups.tpl:227]: Remove unused code.

Comment (cameron-chilton) [reply LEAF_Request_Portal/admin/templates/mod_groups.tpl]: ok removed color statements from function

Comment (cameron-chilton) [reply LEAF_Request_Portal/admin/templates/mod_groups.tpl]: removed entire function, no longer needed.

**Review by abiola-adewuyi — APPROVED** [review]

**Review by g-theriot — APPROVED** [review]

**Pull request merged**